pages/dist/styles.css

```diff
@@ -5,8 +5,11 @@
 .ProseMirror {
   word-wrap: break-word;
   white-space: pre-wrap;
+  white-space: break-spaces;
```

Comment:
What's the reason for the CSS change?

Reply:
Honestly, not sure. This was just added when I ran "npm run build", I'm not sure why it would have done that.

Reply:
Hm, I guess there was something latent in the styling that hadn't been checked in. It shouldn't affect the releases, so no worries!

Reply:
I reset those changes just to be safe. Possibly it should be updated to include them at some point...